l10n_it_fatturapa_in/wizard/wizard_import_fatturapa.py

```diff
@@ -565,14 +565,14 @@ def _addGlobalDiscount(self, invoice_id, DatiGeneraliDocumento):
             self.env['account.invoice.line'].create(line_vals)
         return True
 
-    def _createPayamentsLine(self, payment_id, line, partner_id):
-        PaymentModel = self.env['fatturapa.payment.detail']
-        PaymentMethodModel = self.env['fatturapa.payment_method']
+    def _createPaymentsLine(self, payment_id, line, partner_id):
```

Comment:
@SimoRubi in questo file hai rimosso dei metodi? Ci sono degli altri moduli che usano tali metodi?

Reply:
@eLBati ho rinominato i metodi:

- `_createPayamentsLine`
- `get_e_invoice_lines`
- `get_invoice_line_ids`

che non vengono utilizzati al di fuori di questo modulo